core/src/main/scala/de/heikoseeberger/constructr/Constructr.scala

```diff
@@ -94,10 +95,11 @@ final class Constructr private extends Actor with ActorLogging {
     val ttlFactor           = config.getDouble("constructr.ttl-factor")
     val maxNrOfSeedNodes    = config.getInt("constructr.max-nr-of-seed-nodes")
     val joinTimeout         = getDuration("constructr.join-timeout")
+    val hostPort            = Try(config.getInt("constructr.host-port")).toOption.orElse(cluster.selfAddress.port)
```

Comment:
I think this should go in co-ordination actually. Let me revisit this - just testing it now.